q_yaar_platform/account/models.py

```diff
@@ -0,0 +1,72 @@
+import uuid
+import pghistory
+
+from django.contrib.auth.models import BaseUserManager, AbstractBaseUser
+from django.core.validators import validate_email
+from django.db import models
+
+from common.abstract_models import AbstractExternalFacing, AbstractTimeStamped, AbstractVersioned
+from common.constants import Length
+
+
+class PlatformUserManager(BaseUserManager):
```

Comment:
Overwriting Djangoo's BaseUser and BaseUserManager classes. Extending these to support our custom models. This will make it significantly easier to customize everything.